src/main/java/io/jenkins/plugins/github/checks/api/ChecksDetails.java

```diff
@@ -88,11 +94,25 @@ public ChecksConclusion getConclusion() {
         private final ChecksStatus status;
         private String detailsURL;
         private ChecksConclusion conclusion;
-        private List<Output> outputs = Collections.emptyList();
+        private Output output;
         private List<Action> actions = Collections.emptyList();
 
-        public ChecksDetailsBuilder(final String name, final ChecksStatus status)
-                throws IllegalArgumentException{
+        /**
+         * Construct a builder with the given name and status.
+         *
+         * <p>
+         *     The name will be the same as the check run's name shown on GitHub UI and GitHub uses this name to
+         *     identify a check run, so make sure this name is unique, e.g. "code-coverage".
```

Comment:
I think that human readable names are being used for these checks now? Code coverage, rather than code-coverage

but I assume code is implied here, what other coverage would you be interested in?
```suggestion
         *     identify a check run, so make sure this name is unique, e.g. "Coverage".
```

Reply:
Just copied the name from GitHub doc: https://developer.github.com/v3/checks/runs/#parameters. Either is ok for me. What matters is we may need to find a way to avoid duplicate names.

Reply:
not our issue IMO, other than documenting it